### PR TITLE
more info on Bots

### PR DESCRIPTION
**Pull request by RalfJung**
I'd like to also link to the rust-timer source code and issue tracker, but I was unable to find that. Does anyone know? Cc @Mark-Simulacrum 

It's basically impossible to find the right repo for something these days when it's not rustc, the various rust-lang orgs just have way too many things.^^

### CODE REVIEW

File: src/infra/service-infrastructure.md [hunk 1]
```diff
@@ -25,16 +25,17 @@ The [Rust Log Analyzer](https://github.com/rust-lang/rust-log-analyzer)
...
-([bot user account](https://github.com/bors)) which manages pull requests.
+[Homu](https://github.com/rust-lang/homu/) is a bot often referred to as "bors" due
+to the name of its [bot user account](https://github.com/bors) which manages pull requests.
 Approved pull requests are placed in
 [a queue](http://buildbot2.rust-lang.org/homu/queue/rust) from which tests are
 run.
 
 Documentation on homu commands can be found
 [here](http://buildbot2.rust-lang.org/homu/).
+The homu source code and issue tracker lives [here](https://github.com/rust-lang/homu).
```
Comment (RalfJung) [inline src/infra/service-infrastructure.md]: Is that really the issue tracker to point people to? I do not know.

Comment (RalfJung) [reply src/infra/service-infrastructure.md]: Oh never mind, I only now noticed the first word of the paragraph links to this already.

**Comment by pietroalbini** [comment]
> I'd like to also link to the rust-timer source code and issue tracker, but I was unable to find that. Does anyone know?

https://github.com/rust-lang-nursery/rustc-perf

**Comment by RalfJung** [comment]
@pietroalbini I had found that repo, but the README says nothing about a bot?

**Comment by pietroalbini** [comment]
Well, yeah, but [the bot source is there](https://github.com/rust-lang-nursery/rustc-perf/blob/master/site/src/github.rs).

**Comment by RalfJung** [comment]
Wow, that's very well-hidden. :)

**Comment by RalfJung** [comment]
All right, I added a link to that (and submitted a [PR to rustc-perf](https://github.com/rust-lang-nursery/rustc-perf/pull/485)).

However, I think the bot invocation instructions are also outdated? I heard rumors about a new "try+perf automatically" magic command?

File: src/infra/service-infrastructure.md [hunk 2]
```diff
@@ -62,23 +62,25 @@ wiki](https://github.com/rust-lang/triagebot/wiki) for more information.
...
 and can be browsed on the [perf website](https://perf.rust-lang.org).
 
-One-off performance runs can done by addressing the [rust-timer
-bot](https://github.com/rust-timer). On a PR, do a "try" build (`@bors try`),
-wait for it finish, then leave a comment with the merge commit hash like this:
+One-off performance runs can done by addressing the
+[rust-timer bot](https://github.com/rust-lang-nursery/rustc-perf/blob/master/site/src/github.rs)
+([bot user account](https://github.com/rust-timer)). On a PR, do a "try" build
+(`@bors try`), wait for it finish, then leave a comment with the merge commit
+hash like this:
```
Comment (Mark-Simulacrum) [inline src/infra/service-infrastructure.md]: Could we update this with the new command? Specifically, you should now leave a single comment with something like `@bors try @rust-timer queue`. (Technically, the requirement is that the queue command finishes executing prior to the try command completing successfully).

File: src/infra/service-infrastructure.md [hunk 3]
```diff
@@ -62,23 +62,27 @@ wiki](https://github.com/rust-lang/triagebot/wiki) for more information.
...
+One-off performance runs can done by addressing the
+[rust-timer bot](https://github.com/rust-lang-nursery/rustc-perf/blob/master/site/src/github.rs)
+([bot user account](https://github.com/rust-timer)). You can trigger the
+necessary try-build and queue a perf run by saying
 
-    @rust-timer build ac89e0324b48993d6d148f598ea0e7357e229c79
+    @bors try @rust-timer queue
+
+(Technically, the requirement is that the `queue` command finishes executing prior
+to the `try` command completing successfully.)
```
Comment (Mark-Simulacrum) [inline src/infra/service-infrastructure.md]:
```suggestion
to the try build completing successfully.)
```

**Pull request merged**